docs/AboutUs.md

```diff
@@ -3,10 +3,10 @@
 
 | Display                                             |         Name          |                Github Profile                |               Portfolio                |
 |-----------------------------------------------------|:---------------------:|:--------------------------------------------:|:--------------------------------------:|
-| ![](https://via.placeholder.com/100.png?text=Photo) |     Ningsong Shen     |  [Github](https://github.com/ningsongshen)   | [Portfolio](docs/team/ningsongshen.md) |
-| ![](https://via.placeholder.com/100.png?text=Photo) |  Anne Leong Song En   |    [Github](https://github.com/anneleong)    |  [Portfolio](docs/team/anneleong.md)   |
-| ![](https://via.placeholder.com/100.png?text=Photo) |   Acharya Hiteshri    | [Github](https://github.com/HiteshriAcharya) | [Portfolio](docs/team/benjoburger.md)  |
-| ![](https://avatars.githubusercontent.com/u/90093905?v=4) | Benjamin Goh Guan Rui |   [Github](https://github.com/BenjoBurger)   |   [Portfolio](docs/team/johndoe.md)    |
-| ![](https://via.placeholder.com/100.png?text=Photo) |     Ahmed Shaiyan     |  [Github](https://github.com/AhmedShaiyan)   |   [Portfolio](docs/team/johndoe.md)    |
+| ![](https://avatars.githubusercontent.com/u/12574693?s=400&v=4) |     Ningsong Shen     |  [Github](https://github.com/ningsongshen)   |  [Portfolio](team/ningsongshen.html)   |
+| ![](https://avatars.githubusercontent.com/u/110774636?s=96&v=4) |  Anne Leong Song En   |    [Github](https://github.com/anneleong)    |    [Portfolio](team/anneleong.html)    |
+| ![](https://avatars.githubusercontent.com/u/110652271?s=96&v=4) |   Acharya Hiteshri    | [Github](https://github.com/HiteshriAcharya) | [Portfolio](team/hiteshriacharya.html) |
+| ![](https://avatars.githubusercontent.com/u/90093905?v=4) | Benjamin Goh Guan Rui |   [Github](https://github.com/BenjoBurger)   |   [Portfolio](team/benjoburger.html)   |
```

Comment:
this does not look like benjamin, pls remove!

Reply:
Looks good to me, good work ningsong :)